### PR TITLE
[com_finder] index view: translate content types (solves #10260 - part 1)

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

Solves https://github.com/joomla/joomla-cms/issues/10260 (content types part)

> Load another language than English in back-end.
> Go to Smartsearch Indexed Content equivalent page
> The equivalent filter "- Select type of content -" displays the filter types in English

After PR (example in portuguese):

image
#### Testing Instructions
1. Apply patch
2. Change admin language to other language than en-GB
3. Check the "Content Type" filter in Components -> Smart Search -> Indexed Content

### CODE REVIEW

File: administrator/components/com_finder/models/fields/contenttypes.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		}
+		catch (RuntimeException $e)
+		{
+			JError::raiseWarning(500, $db->getMessage());
+		}
+
+		// Translate.
+		foreach ($contentTypes as $contentType)
+		{
+			$key = FinderHelperLanguage::branchSingular($contentType->text);
```
Comment (mbabker) [inline administrator/components/com_finder/models/fields/contenttypes.php]: There should be something to make sure the class is loaded here (just add a `JLoader::register()` statement to the top of the file).  This isn't an autoloaded class.

Comment (andrepereiradasilva) [inline administrator/components/com_finder/models/fields/contenttypes.php]: you're right! (as usual). forgot that.

**Comment by andrepereiradasilva** [comment]
done

**Comment by infograf768** [comment]
@andrepereiradasilva 

You solved the issue for Indexed but not for Content Maps

screen shot 2016-05-06 at 16 32 43

**Comment by andrepereiradasilva** [comment]
yes, i'm working on that in another PR.
Please mark this one as success.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on c4e507d398ae1dc1dca2b51d556bc9cc2c6f2aee<br /><br />OK for Indexed Content<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10263">issues.joomla.org/joomla-cms/10263</a>.</sub>

**Comment by andrepereiradasilva** [comment]
Check https://github.com/joomla/joomla-cms/pull/10264 for the branch filter in content maps

**Comment by BurtNL** [comment]
I have tested this item :white_check_mark: successfully on c4e507d398ae1dc1dca2b51d556bc9cc2c6f2aee<br /><br />Tested successfully with the German language pack.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10263">issues.joomla.org/joomla-cms/10263</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@infograf768 @BurtNL thanks for testing!

**Comment by infograf768** [comment]
RTC. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10263">issues.joomla.org/joomla-cms/10263</a>.</sub>

**Comment by andrepereiradasilva** [comment]
thanks all for testing and comments!

**Pull request merged**